tests/typecheck/fields/test_related.yml

```diff
@@ -1502,14 +1502,36 @@
 -   case: test_reverse_m2m_relation_checks_other_model
     main: |
         from myapp.models import Author
+        # With builtin manager/queryset
         Author().book_set.filter(featured=True)
         Author().book_set.filter(xyz=True)  # E: Cannot resolve keyword 'xyz' into field. Choices are: authors, featured, id  [misc]
+        reveal_type(Author().book_set.values_list("featured"))  # N: Revealed type is "django.db.models.query.QuerySet[myapp.models.Book, Tuple[builtins.bool]]"
```

Comment:
Rare case, where `Tuple[bool]` and not `Tuple[bool, ...]` is correct.

Reply:
Yeah we're fetching a single column so we get a tuple with a single bool